marklogic-data-hub-central/ui/e2e/cypress/e2e/mastering/e2eMasteringflow.cy.tsx

```diff
@@ -413,9 +415,12 @@ describe("Validate E2E Mastering Flow", () => {
     smPatientArchivedCollection.forEach((cardName: string) => {
       browsePage.verifyCardExist(cardName);
     });
-
-    browsePage.getFacetItemCheckbox("collection", "sm-Patient-archived").click();
+    browsePage.getClearAllFacetsButton().click();
+    browsePage.waitForSpinnerToDisappear();
```

Comment:
![image](https://github.com/marklogic/marklogic-data-hub/assets/126084579/a72d3094-2e43-49e9-97e6-eaa8ef4737bd)

Let's use this method, please